tests/ConcordiumNetSdk.UnitTests/EmbeddedResourceDataAttribute.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using Xunit.Sdk;
+
+/// <summary>
+/// Data attribute which allows to capture data of resources embedded
+/// in the manifest for use in tests.
```

Comment:
I don't understand this comment. What is the "data attribute"?

Reply:
That's the Xunit lingo for a data source for a data theory. It refers to the super class ("is-a"). I will link it in this doc, however since it may be confusing.

Reply:
Ok, thanks.